### PR TITLE
[stable-1.1.x] Bump chart version to 0.8.44

### PR DESCRIPTION
**Pull request by gracedo**
**What type of PR is this?**
Chore

**What this PR does/ why we need it**:
Bump chart to 0.8.44 to be released in patch version 1.1.2. In this version, yakcl is bumped to v0.3.3 which is required to be compatible with konvoy 1.5.2

**Which issue(s) this PR fixes**:

no issue
**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Checklist**

- [x] The commit message explains the changes and why are needed.
- [ ] The code builds and passes lint/style checks locally.
- [ ] The relevant subset of integration tests pass locally.
- [ ] The core changes are covered by tests.
- [ ] The documentation is updated where needed.

### CODE REVIEW

**Comment by dkoshkin** [comment]
Looking for the federation webhook not to fail [here](https://teamcity.mesosphere.io/buildConfiguration/ClosedSource_MkeLite_MkeLiteE2e?branch=dkoshkin%2Fcontainerd-v1.3.7-1.5&buildTypeTab=overview&mode=builds)

**Comment by gracedo** [comment]
@dkoshkin this error? :/
```
Error: failed to push the cluster state: error updating the konvoy cluster: admission webhook "webhook.federation.kommander.mesosphere.io" denied the request: spec.cluster.containerRuntime.containerd.version: Invalid value: "1.3.7": must be within range: 1.2.6 || 1.2.13 || 1.3.4
```

**Comment by dkoshkin** [comment]
> @dkoshkin this error? :/
> 
> ```
> Error: failed to push the cluster state: error updating the konvoy cluster: admission webhook "webhook.federation.kommander.mesosphere.io" denied the request: spec.cluster.containerRuntime.containerd.version: Invalid value: "1.3.7": must be within range: 1.2.6 || 1.2.13 || 1.3.4
> ```

Yeah :( 

Where is this image coming from? 🤔 
```
image: mesosphere/kommander-federation-webhook:v0.2.0
```

**Comment by dkoshkin** [comment]
Ok removed `1.2.0` dir https://github.com/mesosphere/kubeaddons-kommander/pull/141/commits/18992d7118010ab0ee386cd2f943a76a48ec9390 and retriggered the test https://teamcity.mesosphere.io/buildConfiguration/ClosedSource_MkeLite_MkeLiteE2e/2850975

File: addons/kommander/1.1.2/kommander-1.yaml [hunk 1]
```diff
@@ -6,8 +6,8 @@ metadata:
   labels:
     kubeaddons.mesosphere.io/name: kommander
   annotations:
-    catalog.kubeaddons.mesosphere.io/addon-revision: "1.2.0-2"
-    appversion.kubeaddons.mesosphere.io/kommander: "1.2.0-beta.0"
+    catalog.kubeaddons.mesosphere.io/addon-revision: "1.1.2-1"
```
Comment (shaneutt) [inline addons/kommander/1.1.2/kommander-1.yaml:9]: There is a problem here: using an OLDER revision version for a newer revision may result in clusters being unable to upgrade properly, we will need to do something about this in order to proceed :thinking:

Comment (dkoshkin) [reply addons/kommander/1.1.2/kommander-1.yaml:9]: Its just showing up like this because of the way Grace copied the existing yaml from `v1.2.0` these 2 lines and the `version` are the only difference in `1.1.1` aswell. 

TL;DR the resulting file is the same.

Comment (shaneutt) [reply addons/kommander/1.1.2/kommander-1.yaml:9]: I'm not sure... the way this PR is working currently, it's effectively _removing_ the latest 1.2 revision completely, is it not?

Comment (gracedo) [reply addons/kommander/1.1.2/kommander-1.yaml:9]: 1.2 revision should not have made its way into this branch in the first place, which is why we are removing it now - this is a `stable-1.1.x` branch used for patches on 1.1, so this branch should not contain any revisions > 1.1.x

Comment (shaneutt) [reply addons/kommander/1.1.2/kommander-1.yaml:9]: Ah, got it.
